README.md

```diff
@@ -601,27 +603,6 @@ You may also add the steps above in your existing workflows. The generated code
 
 For more information on how you can use GitHub Actions with Clarinet, please see the [A Simple CI With Clarinet and GitHub](https://www.youtube.com/watch?v=cEv6Mi4EcKQ&list=PL5Ujm489LoJaAz9kUJm8lYUWdGJ2AnQTb&index=8) YouTube video
 
-### Extensions
-
-Clarinet can easily be extended by community members: open source contributions to Clarinet are welcome, but you may also write your own Clarinet extensions if you want to integrate Clarity contracts with your own tooling and workflow.
-
-| Name                      | wallet access | disk write | disk read | Deployment                                                            | Description                                                                                                                                       |
-| ------------------------- | ------------- | ---------- | --------- | --------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
-| stacksjs-helper-generator | no            | yes        | no        | https://deno.land/x/clarinet@v1.0.2/ext/stacksjs-helper-generator.ts | Facilitates contract integration by generating some typescript constants that can be used with stacks.js. Never hard code a stacks address again! |
-|                           |               |            |           |                                                                       |
-
-#### How to use extensions
-
-Extensions are run with the following syntax:
-
-```
-clarinet run --allow-write https://deno.land/x/clarinet@v0.29.0/ext/stacksjs-helper-generator.ts
-```
-
-An extension can be deployed as a standalone plugin on Deno, or may also be a local file if it includes sensitive / private setup informations.
-
-As illustrated in the example above, permissions (wallet / disk read / disk write) are declared using command flags. If at runtime, the Clarinet extension is trying to write to disk, read disk, or access wallets without permission, the script will fail.
-
```

Comment:
Fully removing this from the readme in an attempt to actually make it smaller instead of bigger.

`extensions` and `clarinet run` are really not used a lot so I don't think it's worth mentioning it (and adding deprecation warning) here